examples/platform/silabs/provision/ProvisionStorageFlash.cpp

```diff
@@ -789,7 +789,7 @@ CHIP_ERROR Storage::DecryptUsingOtaTlvEncryptionKey(MutableByteSpan & block, uin
     return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
 #endif // SL_MBEDTLS_USE_TINYCRYPT
 }
-#endif // OTA_ENCRYPTION_ENABLE
+#endif // SL_MATTER_ENABLE_OTA_ENCRYPTION
```

Comment:
doesn't the ProvisionStorageFlash.cpp also need a CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE implementation of the few  `OtaTlvEncryptionKey` functions